src/ui/components/TransferGrain.js

```diff
@@ -18,102 +18,192 @@ export type Props = {|
+                  name="amount"
+                  min="0"
+                  placeholder="(amount)"
+                  value={amount !== "0" ? amount : ""}
+                  onChange={(e) => setAmount(e.currentTarget.value)}
+                />
+              </div>
+              <span>
+                {sourceAccount &&
+                  sourceAccount.identity &&
+                  ` max: ${format(sourceAccount.balance, 2)}`}
+              </span>
+            </div>
+            <div className={classes.triangle} />
+          </div>
+          <div
+            className={`${classes.dropdown} ${classes.centerRow} ${classes.third}`}
+          >
+            <AccountDropdown
+              ledger={ledger}
+              setCurrentIdentity={setReceiver}
+              placeholder="To..."
+            />
+          </div>
+        </div>
+        <div className={classes.centerRow}>
+          <Input
+            className={classes.memo}
+            disableUnderline
             type="text"
             name="memo"
             value={memo}
+            placeholder="Memo"
             onChange={(e) => setMemo(e.currentTarget.value)}
           />
-        </p>
-        <input
-          disabled={
-            !Number(amount) ||
-            !(sourceIdentity && destIdentity) ||
-            gt(fromFloatString(amount), maxAmount)
-          }
-          type="submit"
-          value="transfer grain"
-        />
-        <br />
-        <input
-          type="button"
-          value="save ledger to disk"
-          onClick={() => {
-            fetch("data/ledger.json", {
-              headers: {
-                Accept: "text/plain",
-                "Content-Type": "text/plain",
-              },
-              method: "POST",
-              body: ledger.serialize(),
-            });
-          }}
-        />
-      </form>
-    </div>
+        </div>
+        <div className={classes.centerRow}>
+          <Button
+            className={`${classes.centerRow} ${classes.half}`}
+            style={{background: "#6558F5"}}
+            variant="contained"
+            type="submit"
+            disabled={
+              !Number(amount) ||
+              !(
+                sourceAccount &&
+                sourceAccount.identity &&
+                destAccount &&
+                destAccount.identity
+              ) ||
+              gt(fromFloatString(amount), sourceAccount.balance)
+            }
+          >
+            transfer grain
+          </Button>
+          <Button
+            className={`${classes.centerRow} ${classes.half}`}
+            style={{background: "#C9CED2"}}
+            variant="contained"
+            onClick={() => {
+              fetch("data/ledger.json", {
+                headers: {
+                  Accept: "text/plain",
+                  "Content-Type": "text/plain",
+                },
+                method: "POST",
+                body: ledger.serialize(),
+              });
+            }}
```

Comment:
We should abstract over this pattern so that we create a `persistLedger` method once at the app top level and then pass the function down. No change requested in this PR, it's a separate cleanup, just noting that this is violating DRY.